components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -145,42 +145,6 @@
 
 <!-- Bulk aerosol physical properties (includes optics) -->
 
-<!-- CAM3 Bulk aerosols.  Optics for CAM-RT -->
```

Comment:
Do you know if we would need special optics files if we were to attempt to revive the BAM option and run it with RRTMG?

Reply:
Most likely, but they may exist already. Using CAM-RT optics files would not be an option I don't think, I believe the bands are different.